packages/web3-eth/test/integration/web3_eth/call.test.ts

```diff
@@ -0,0 +1,79 @@
+		const transaction: Transaction = {
+			from: accounts[0],
+			data: greetContractDeploymentData,
+			gas: '0x740b8',
+		};
+		const response = await web3Eth.sendTransaction(transaction);
+		greeterContractAddress = response.contractAddress as string;
+	});
+
+	afterAll(() => {
+		if (getSystemTestProvider().startsWith('ws')) {
+			(web3Eth.provider as WebSocketProvider).disconnect();
+		}
+	});
+
+	it('should make a call to deployed Greeter contract', async () => {
+		const expectedEncodedGreet =
+			'0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000017736f6c79656e7420677265656e2069732070656f706c65000000000000000000';
+		const expectedDecodedGreet = 'solyent green is people';
+		const greetCallData = '0xcfae3217';
+		const transaction: TransactionCall = {
+			to: greeterContractAddress,
+			data: greetCallData,
+		};
+		const response = await web3Eth.call(transaction);
```

Comment:
We should also test it without using defaultBlock and passing it explicitly, and also with modified web3.eth.defaultBlock

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5056/commits/01d49e28028a51392582c70b53b08f2a8507cdaa) commit

Reply:
@jdevcs  Regarding modifying the defaults of `Web3Context`, I think that's better handled by explicit tests that @avkos is working on, instead of including these tests for every RPC method in `Web3Eth`